client/my-sites/site-settings/jetpack-sync-panel/index.jsx

```diff
@@ -22,8 +22,7 @@ const debug = debugModule( 'calypso:site-settings:jetpack-sync-panel' );
 const SYNC_STATUS_ERROR_NOTICE_THRESHOLD = 3; // Only show sync status error notice if >= this number
 
 class JetpackSyncPanel extends Component {
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillMount() {
+	componentDidMount() {
```

Comment:
There should also be a `componentDidUpdate` that fetches on `siteId` change.

Reply:
Good catch! I've addressed that in 41fc5d7d760ee9e218c08fea84f38f13ab054b74.